src/js/js-loader.ts

```diff
@@ -32,9 +32,10 @@ function jsLoader(source: string, sourceMaps: any): void {
 }
 
 function extractTranslations(loader: TranslateLoaderContext, source: string, sourceMaps: any): void {
+    const options = parseOptions(loader.version, loader.query);
     loader.pruneTranslations(loader.resource);
 
-    const visitor = new TranslateVisitor(loader);
+    const visitor = new TranslateVisitor(loader, options);
```

Comment:
Can we just pass `options.parserOptions` instead. Makes it easier to separate the options in case we add new ones in the future.